index.html

```diff
@@ -4,9 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="./styles/index.css" rel="stylesheet" type="text/css">
+	<script src="./src/index.js"></script>
+	<script src="./node_modules/axios/dist/axios.min.js"></script>
```

Comment:
You may see `script` elements commonly placed under the closing `body` tag in order to ensure that the DOM content is loaded first before any scripts are run. 

~~Since you are not registering your event handlers under a`DOMContentLoaded` event listener, it's important to place your `script` elements at the bottom.~~ I see now that you did make sure the DOM loaded first on line 12